.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -29,7 +29,7 @@ React Native version:
 Describe what you expected to happen:
 
 
-Snack, code example, or link to a repository:
+[Snack](https://snack.expo.io/), screenshot, code example, or link to a repository:
```

Comment:
This should be covered by the comment below (Line 36). I did not use an inline link intentionally, as the bug report template is commonly seen in plain text within the "new issue" text area, and I want to make sure this one-liner is easy to read. Any additional details can go in the commented block of text that follows.

Reply:
I am ok with adding "screenshot" to the list here, if it comes with a clarification in the comment block below that states the screenshot should be used for bugs of a visual nature. The situation I would like to avoid is getting bug reports with screenshots of errors / log spew.

Reply:
Thank you very much, I did not realize that I had already been added in a comment. LGTC!